llvm/include/llvm/ProfileData/SampleProf.h

```diff
@@ -502,6 +497,12 @@ struct SampleContextFrame {
     }
     return OContextStr.str();
   }
+
+  uint64_t getHashCode() const {
+    uint64_t NameHash = FuncName.getHashCode();
+    uint64_t LocId = Location.getHashCode();
+    return NameHash + (LocId << 5) + LocId;
+  }
```

Comment:
I already see a `hash_value` overload a couple lines below (line 508). Do we need this? If yes, then we should find a better name and a comment in how this is different from the `hash_value()` overload.